src/utils/mev.ts

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios'
+import { ethers } from 'ethers'
+const logger = require('../services/logger')
+
+const bevUrl = 'https://bev-relay.kolibr.io'
```

Comment:
this should be on a environment variable IMO, not only for privacy but to quickly change the URL if needed